packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sei-js/react",
-  "version": "1.2.0",
+  "version": "0.0.0-internal-20230316005727",
```

Comment:
Could you undo the local changeset changes? This version should only be for testing and not be committed to the repo 

you might also need to undo the changelog changes and changes in `.changeset` as well